NetKAN/LaunchPadLamp.netkan

```diff
@@ -0,0 +1,10 @@
+{
+    "spec_version" : "v1.0",
```

Comment:
"spec_version" : "v1.4",

Reply:
spec_version "v1.0" is not actually valid. I'm not sure why it isn't complaining, but for this mod it should be one of the following lines:

    "spec_version" : "v1.4",
    "spec_version" : 1,